generator/generator.go

```diff
@@ -132,6 +130,30 @@ type Options struct {
 	LocalPrefix string
 }
 
+type methodsList map[string]Method
+
+type processInput struct {
+	fileSet        *token.FileSet
+	currentPackage *packages.Package
+	pacakge        *ast.Package
+	targetName     string
+	genericParams  genericParams
+}
+
+type targetProcessInput struct {
+	processInput
```

Comment:
This field doesn't seem to be used anywhere.

Reply:
the processInput is used on `findTarget` function.